designs/virtualization/4.x/create-vm-dropdowns/create-vm-dropdowns.md

```diff
@@ -0,0 +1,112 @@
+Server is the default
+
+![Create VM - General - flavor](img/1-1-2.jpg)
+Flavor options
+Medium is the default
+
+![Create VM - General](img/1-2-1.jpg)
+'Boot operating system via' field options
+
+![Create VM - General](img/1-2.jpg)
+Selecting an option from the 'Boot operating system via' field will expose below all the fields required to connect the VM to an OS image source.
+For example: on selecting the 'Existing PVC' option, additional fields will appear below, required to select a PVC available on the cluster.
+The disk name should be added here as well to establish a better link between this step and the Storage step.
+An info alert will notify the user that a disk will be added to to the disks list on the Storage step.
+
+![Create VM - OS disk on list](img/1-3.jpg)
+Within the storage step, the disk added on the general step will appear as the first item on the list.
+This disk cannot be deleted but can be modified.
+
+![Create VM - OS disk modal](img/1-3-1.jpg)
+Editing that disk will display a warning notification within the edit modal.
+
+## Netwroking step
+
+![Create VM - Networking -list](img/2-0.jpg)
+NIC list.
+For any VM source selection (with the exception of PXE) a default NIC will be added.
+This NIC exposes the VM to the cluster.
+
+![Create VM - Networking -Default](img/2-1.jpg)
+On editing this default NIC, an info alert will remind the user that modifying this NIC might break that 'exposing to cluster' configuration.
+
+![Create VM - Networking -Add](img/2-2.jpg)
+Add NIC modal.
+The Advanced drawer is closed by default, as seen in the previous image.
+All fields have moved into the advanced drawer, except the Name and Network fields.
+Network field does not require description for each option since these options are created and managed by the user.
+
+![Create VM - Networking -Model](img/2-3.jpg)
+'Model' dropdown options.
+ViritIO marked as (recommended)
+
+![Create VM - Networking -Binding](img/2-4.jpg)
+'Binding method' dropdown options (previously 'Type')
+
+## Storage
+
+![Create VM - Storage -list](img/3-0.jpg)
+Disks list.
+In case no disk type OS image source selected in the General step, a default disk will be created in order to make the VM operational.
+
+![Create VM - Storage -Default disk](img/3-0-0.jpg)
+Editing the default disk will present an info box notification.
+
+![Create VM - Storage -list](img/3-1.jpg)
+The Add/Edit modal has been modified.
+- Name
+- Type changed to 'Disk content'. This will change the configuration below
+- Advanced drawer added containing: 
+    - Interface
+    - Volume mode
+    - Access mode (should be automatically selected and locked to the Storage class default)
+    - Make disk bootable checkbox (new)
+
+![Create VM - Storage -list](img/3-2.jpg)
+Disk content dropdwon options.
+
+## Advanced
+
+ ![Create VM - Advanced -drawers](img/4-0.jpg)
+The advanced step drops the sub-steps from the wizard navigation in favor for section drawers within the step body.
```

Comment:
This will give a quite headache for users which want to customize it. 

- you have to click multiple times to explore all the options
- the page will get very long once you do that.
- there will be double scrolling for the Cloud Init editor
- the Cloud Init editor might get more complicated in the future and it might overwhelm the users
- the boot order might get used quite often by the users

Could we solve the original problem by adding an `Skip Advanced` button to the wizard footer to be more explicit ? We could assign this to a primary button and the `Next` would be secondary.

thoughts? 

Nevertheless the users already have an option of choosing `Review and create` option .

Reply:
That's a good point @suomiy 
I know we've done some investigation around advanced in the past [here](https://docs.google.com/document/d/1zGYFLJ3PVa-q4smMIiXH8tolLdUq8A5vn5Ybf9OhF6g/edit#heading=h.36pfaes06mcj)
@Ranelim it might be a good idea to pull this from this design and have a separate effort to explore this.

Reply:
The problem of having to go through all the sub-steps is quite the opposite of what an "Advanced" step should do. We want to avoid exhausting the user. Adding a "Review and Create" link button is not in the main flow and it will not solve the fact that we make the user go through literally everything.
Plus, the points mentioned above are actually similar or even identical to the same issues we get with the sub-steps. 

> it might be a good idea to pull this from this design and have a separate effort to explore this.
Sure thing